python/ffsim/testing/testing.py

```diff
@@ -153,3 +153,21 @@ def assert_allclose_up_to_global_phase(
         err_msg=err_msg,
         verbose=verbose,
     )
+
+
+def interaction_pairs_spin_balanced(
```

Comment:
I've added this function to `variational.util`, see https://github.com/qiskit-community/ffsim/pull/344